drivers/scsiutil.py

```diff
@@ -644,6 +644,11 @@ def sg_readcap(device):
         # retry one time for "Capacity data has changed"
         (rc,stdout,stderr) = util.doexec(readcapcommand)
     if rc != 0:
-        raise util.SMException("util.sg_readcap(%s) failed" % (device))
-    (blockcount,blocksize) = stdout.split()
+        raise util.SMException("scsiutil.sg_readcap(%s) failed" % (device))
+    match = re.search('(^|.*\n)(0x[0-9a-fA-F]+) (0x[0-9a-fA-F]+)\n$', stdout)
+    if not match:
+        raise util.SMException("scsiutil.sg_readcap(%s) failed to parse: %s"
+                               % (device, stdout))
+    blockcount = match.group(2)
+    blocksize = match.group(3)
```

Comment:
@robertbreker Could we shorten this to something like below ?
`(blockcount, blocksize) = match.group(2, 3)`